mypy.ini

```diff
@@ -0,0 +1,27 @@
+[mypy]
+# Be vewy, vewy stwict, we'we hunting wabbits^Wbugs.
+strict = True
+# Automagically make imported object part of mypy's view of module's namespace.
+implicit_reexport = True
+# Mypy will install packaged type stubs when possible.
+install_types = True
+non_interactive = True
+
+files = tmt/utils.py
+
+# This is probably temporary: don't follow imports to other tmt packages
+# and modules, but once the list of files is complete, this should be dropped
+# in favor of simply checking whole tmt package.
+follow_imports = skip
```

Comment:
Just so that I understand this correctly, once we have all the modules typed, this will be removed and `files` above will be adjusted (probably to just `tmt/` right? or do we have to list all the files?)

Reply:
Correct. We do not need to list or files, mypy can deal with files, directories, and modules or packages at the same time, therefore `tmt` (a package) or `tmt/` (a directory) should provide the desired effect.